### ISSUE TITLE
Expose reference to the underlying scroller node

### ISSUE THREAD

**Issue by fabiospampinato** [petyosi/react-virtuoso#262]
It'd be useful if a Virtuoso instance exposed a reference to the underlying DOM node for the scroller component.

Reasons for doing that:

- Without it one needs to provide a custom Scroller component just to get a reference to it.
  - In order to provide a custom Scroller component first of all one needs to use `forwardRef`, which makes it more complicated to adopt Virtuoso, plus it's kinda useless really, this is the first time I've had to use `forwardRef`, which btw will probably be removed [eventually](https://twitter.com/dan_abramov/status/1054960391955120128?lang=en).
    - Lastly you can't just pass your custom ref to the scroller's div now, because Virtuoso wants one too, so you need to create a new ref that merges the two you need to attach and pass that on to the component instead, which is not exactly difficult to do but also not something that pretty much anybody should need to ever think about.

Basically I'd just expose Virtuoso's refs, at least for the scroller component, and I would personally stop requiring `forwardRef` too as it really just makes the library harder to adopt for no benefit whatsoever (that I can see anyway).

**Comment by petyosi** [comment]
Love the ref exposure suggestion. Question: what are you trying to achieve? So far, I am aware of people trying to integrate Virtuoso with a custom scroller library or with Drag & Drop one. I would like to have a use case I can test the ergonomics against.

**Comment by fabiospampinato** [comment]
I need a reference to the scroller in particular to control scrolling.

The provided APIs for that are a bit too low-level, like for example `scrollToIndex` allows you to scroll to the start/end/center, but without knowing where exactly you are going to scroll to you can't always pick one of those modes. E.g. if the item to scroll to is fully within the viewport the list shouldn't scroll at all.

Also my "Tree" component can work with a Virtuoso list or a fixed list, by manipulating the DOM directly I only have to code the logic for that once, e.g. I'm not relying on any Virtuoso-specific APIs.

**Comment by petyosi** [comment]
Makes sense, thank you.
